tests/Tests/200__monitor_and_manage/200__metrics/204_check_pager_duity.robot

```diff
@@ -0,0 +1,49 @@
+*** Settings ***
+Documentation   RHODS_Pager_Duity_key__VERIFICATION
```

Comment:
There is a typo in the filename and other places of this file (it's "pager duty", not "duity")